drivers/bluetooth/hci/nrf53_support.c

```diff
@@ -30,16 +29,15 @@ int bt_hci_transport_teardown(const struct device *dev)
 int bt_hci_transport_setup(const struct device *dev)
 {
 	ARG_UNUSED(dev);
-#if !defined(CONFIG_TRUSTED_EXECUTION_NONSECURE) || defined(CONFIG_BUILD_WITH_TFM)
```

Comment:
GPIO setup may fail? (but will go unnoticed, `DEBUG_SETUP` may not be covered in CI.

Reply:
`DEBUG_SETUP()` is now a macro expanding to a set of unchecked calls, so as it is designed, can't fail.